specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/workspace.json

```diff
@@ -1326,7 +1326,8 @@
           "type": "boolean",
           "x-ms-mutability": [
             "create",
-            "read"
+            "read",
+            "update"
           ]
```

Comment:
Just remove x-ms-mutability.

Reply:
Removing this raises alerts in S360 for ROUNDTRIP_INCONSISTENT_PROPERTY

Reply:
If that's the case then I doubt the current code will fix it. Can you share a link to the S360 issue?